File: api/webapi/Dtos/CtdonDh/CtdonDhWithTenSp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace webapi.Dtos.CtdonDh
{
    public class CtdonDhWithTenSp
    {
        public string MaDdh { get; set; } = null!;

        public string MaSp { get; set; } = null!;
        public int? GiaBan { get; set; }

        public string TenSp { get; set; } = null!;

        public int? SoLuong { get; set; }
    }
}

File: client/quan_ly_ban_hang/src/main/java/com/mycompany/Dtos/CtdonDh/CtdonDhWithTenSp.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.Dtos.CtdonDh;

/**
 *
 * @author ACER
 */
public class CtdonDhWithTenSp {
    private String maDdh;
    private String maSp;
    private String tenSp;
    private Integer giaBan;
    private Integer soLuong;

    public Integer getGiaBan() {
        return giaBan;
    }

    public void setGiaBan(Integer giaBan) {
        this.giaBan = giaBan;
    }

    // Constructor
    public CtdonDhWithTenSp() {
    }

    // Getters and Setters
    public String getMaDdh() {
        return maDdh;
    }

    public void setMaDdh(String maDdh) {
        this.maDdh = maDdh;
    }

    public String getMaSp() {
        return maSp;
    }

    public void setMaSp(String maSp) {
        this.maSp = maSp;
    }

    public String getTenSp() {
        return tenSp;
    }

    public void setTenSp(String tenSp) {
        this.tenSp = tenSp;
    }

    public Integer getSoLuong() {
        return soLuong;
    }

    public void setSoLuong(Integer soLuong) {
        this.soLuong = soLuong;
    }
}

